back_end/tweeter/test/tweeter_web/controllers/page_controller_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule TweeterWeb.PageControllerTest do
```

Comment:
We dont need this test file , since we are removing the page controller related files